shared/src/test/resources/ngsild/events/entityCreateEvent.jsonld

```diff
@@ -1,5 +1,8 @@
 {
     "entityId":"urn:ngsi-ld:Vehicle:A4567",
     "operationPayload":"{\n \"id\": \"urn:ngsi-ld:Vehicle:A4567\",\n\"type\": \"Vehicle\",\n\"brandName\": {\n\"type\": \"Property\",\n\"value\": \"Mercedes\"\n},\n\"@context\": [\n\"http://example.org/ngsi-ld/latest/vehicle.jsonld\",\n\"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld\"\n]\n}",
```

Comment:
better to remove the @contexts in the `operationPayload` (or `updatedEntit`) in all the sample payloads in the tests (to avoid misunderstandings)